ovirt_imageio/client/_ui.py

```diff
@@ -75,19 +75,11 @@ def close(self):
 
     def _draw(self, last=False):
         elapsed = self._now() - self._start
+        progress = f"{self._value:3d}%" if self.size else "----"
```

Comment:
As size can now be `None`, maybe here we want to use `if self.size is not None` instead?

Reply:
But caller can use size=0, and this will try to divide by zero.